src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,57 +1,165 @@
-import React from 'react';
+import React, { useEffect, useRef, useState } from 'react';
+import cn from 'classnames';
+import { Movie } from '../../types/Movie';
 import './FindMovie.scss';
+import { getMovie } from '../../api';
+import { MovieData } from '../../types/MovieData';
+import { MovieCard } from '../MovieCard';
+
+type Props = {
+  onAdd: (movie: Movie) => void;
+};
+
+export const FindMovie: React.FC<Props> = ({ onAdd }) => {
+  const [movie, setMovie] = useState<Movie | null>(null);
+  const [searchFieldValue, setSearchFieldValue] = useState<string>('');
+  const [searchName, setSearchName] = useState<string>('');
+  const [isSearching, setIsSearching] = useState<boolean>(false);
+  const [isError, setIsError] = useState<boolean>(false);
+  const searchField = useRef<HTMLInputElement | null>(null);
+
+  const handlerChangeSearchField = (event: HTMLInputElement) => {
+    if (event.value.length > 0) {
```

Comment:
The argument type of the 'handlerChangeSearchField' function should be 'React.ChangeEvent<HTMLInputElement>' instead of 'HTMLInputElement'.